filesystems/gcs/src/main/scala/cromwell/filesystems/gcs/batch/GcsBatchIoCommand.scala

```diff
@@ -75,7 +75,7 @@ case class GcsBatchCopyCommand(
                               )
   extends IoCopyCommand(source, destination) with GcsBatchIoCommand[Unit, RewriteResponse] {
   override def commandDescription: String = s"GcsBatchCopyCommand source '$source' destination '$destination' " +
-    s"setUserProject '$setUserProject'"
+    s"setUserProject '$setUserProject' rewriteToken '$rewriteToken'"
```

Comment:
Is `rewriteToken` sensitive? My quick glance at some related code makes me think not, but I didn't dig deep enough to determine for sure.

Reply:
Not that I'm aware of.

Reply:
More info jic anyone comes across this thread later:

Here are the docs regarding [rewrites and tokens](https://cloud.google.com/storage/docs/json_api/v1/objects/rewrite). From my understanding of the docs there are no warnings about rewrite tokens being exploitable in some way. I think of them sort of like [query cursors](https://cloud.google.com/firestore/docs/query-data/query-cursors), but used for tracking copy progress. I don't see anything where the tokens may be used in any way to modify or expose the contents of the file being duplicated.

Reply:
Thanks. I figured it was something like that, but the word "token" caught my attention.